.github/dependabot.yml

```diff
@@ -17,3 +17,11 @@ updates:
       - "area/dependency"
       - "release-note-none"
       - "ok-to-test"
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
Thanks, I think we can only use the `/images` directory.
Only the Dockerfile in this directory needs to be automatically updated